### ISSUE TITLE
(MW/feature request) manual reverse gear like in PS2 demo/Carbon?

### ISSUE THREAD

**Issue by ToruTheRedFox** [xan1242/NFS-XtendedInput#6]
it's a controls mod so maybe it'll fit

getting parity with final console release is priority tho

**Comment by xan1242** [comment]
Yeah that is indeed interesting that it doesn't shift down to reverse even though it should technically.

Internally the gears are enumerated, so it should be very easy to just make it decrement the gear value one more step.

Gear -1 = Reverse
Gear 0 = Neutral
Gear 1 = Gear 
... and so on

So it's just a matter of finding the joy handler code for that (which is PlayerInput I think), which I already did, so it shouldn't be a too hard fix.

**Comment by ToruTheRedFox** [comment]
forgot that Carbon on console also has reverse on brake pedal regardless of transmission setting
